superset/examples/configs/datasets/examples/users_channels-uzooNNtSRO.yaml

```diff
@@ -73,4 +73,4 @@ columns:
   description: null
   python_date_format: null
 version: 1.0.0
-database_uuid: 566ca280-3da8-967e-4aa4-4b349218736a
+database_uuid: a2dc77af-e654-49bb-b321-40f6b559a1ee
```

Comment:
Other datasets inside `superset/examples/configs/datasets/examples` already had the UUID set to this, so I kept it and updated this one.